### ISSUE TITLE
[Bug][a11y]: MenuItem with submenu lacks aria-haspopup and aria-expanded

### ISSUE THREAD

**Issue by majornista** [adobe/spectrum-web-components#3248]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

MenuItem

### Expected behavior

A MenuItem containing a submenu should have `aria-haspopup="true"`, `aria-expanded` to reflect the expanded/collapsed state of the submenu, and `aria-controls` with the IDREF for the expanded menu.

### Actual behavior

MenuItem provides no indication that the menu expands to a submenu, and no indication of the current state.

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error
